Scroller/EditableWrapper.js

```diff
@@ -623,7 +623,7 @@ const EditableWrapper = (props) => {
 			mutableRef.current.needToPreventEvent = true;
 		} else if (is('left', keyCode) || is('right', keyCode)) {
 			if (selectedItem) {
-				if (mutableRef.current.lastKeyEventTargetElement?.getAttribute('role') !== 'button') {
+				if (mutableRef.current.lastKeyEventTargetElement?.getAttribute('role') !== 'button' && target.getAttribute('role') !== 'button') {
```

Comment:
I found one another issue seems to be caused by `lastKeyEventTargetElement` related condition.
Reproduciable Step
1. Start editing by clicking on an item
2. Move the spotlight to the upper button
3. Press left or right key
4. Move the cursor to the icon item
5. Press left or right key
Expected result: The item moves left or right
Actual result: The item doesn't move

I think we could remove the fore condition.
And as we discussed, let's dicuss with UX team for the clearer scenario for this.
e.g. How many buttons can be in the upper area? How should spotlight move from the last button when we press right key, etc.

Reply:
It's fixed.